apps/model-ad/app/project.json

```diff
@@ -101,7 +101,8 @@
           "buildTarget": "model-ad-app:build:development"
         }
       },
-      "defaultConfiguration": "development"
+      "defaultConfiguration": "development",
+      "continuous": true
```

Comment:
This is one of the updates brought by Nx 21. This indicate that the task is expected to keep running.